_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -0,0 +1,723 @@
+---
+title: How to achieve compliance with the CIS AWS Foundations Benchmark
+categories: Compliance
+image: /assets/img/guides/cis-compliance/cis-logo.png
+excerpt: Learn how to achieve and maintain compliance with each of the CIS AWS Foundations Benchmark recommendations.
+tags: ["aws", "security", "compliance"]
+cloud: ["aws"]
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+This is a complete guide to help you achieve compliance with the
+link:https://www.cisecurity.org/benchmark/amazon_web_services/[CIS AWS Foundations Benchmark] using
+link:https://gruntwork.io/infrastructure-as-code-library/[the Gruntwork Infrastructure as Code Library] in
```

Comment:
The guides are designed so the first ~2/3rds—the core concept and prod-grade design—are usable by anyone, and not just Gruntwork customers. So perhaps don't mention the IaC Lib until the 3rd part, the deployment walkthrough?

Reply:
Also, should we call out right at the top which version of the standard this is for?